Brokkr/src/main/cpp/commands/IntakeEjectObject.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "commands/IntakeEjectObject.h"
+
+#include <frc2/command/InstantCommand.h>
+#include "commands/IntakeSpinSimple.h"
+#include <frc2/command/WaitCommand.h>
```

Comment:
Include frc2 includes before local includes. Move this line above the previous line.